tests/rspack-test/__snapshots__/Config.test.js.snap

```diff
@@ -0,0 +1,81 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`config configCases/source-map/extract-source-map should pass 1`] = `
+Array [
+  webpack:///./external-source-map.txt,
+  webpack:///./extract2.js,
```

Comment:
This is a little different with webpack.

We don't have:

```
  "webpack:///external node-commonjs \\"fs\\"",
  "webpack:///external node-commonjs \\"path\\"",
  "webpack:///webpack/bootstrap",
```

See: https://github.com/webpack/webpack/pull/19824/files#diff-de61f5c1c819fec0a01a522d41e75824ba8e13f758920980349790b701ebe95cR11339